admin/src/pages/CreationConfirm.spec.js

```diff
@@ -399,6 +436,10 @@ describe('CreationConfirm', () => {
             await wrapper.find('a[data-test="all"]').trigger('click')
           })
 
+          it('refresh the currentPage to 1', () => {
+            expect(wrapper.vm.currentPage).toBe(1)
+          })
+
```

Comment:
After the next it block, add a new describe block and test the pagination there (we have most items in all tab). Then, in another describe, click back to the open tab to check, that currentPage is really reseted